### PR TITLE
Cb 18

### PR DESCRIPTION
**Pull request by CalebMacdonaldBlack**

### CODE REVIEW

File: resources/migrations/20170222203502-aggregates.up.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE aggregates (
+  id          UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+  description TEXT             NOT NULL,
+  activities  UUID REFERENCES activities (id)
```
Comment (lrobert142) [inline resources/migrations/20170222203502-aggregates.up.sql]: Type required to be a postgres Array type

**Review by lrobert142 — DISMISSED** [review]
See comments

File: resources/migrations/20170222203502-aggregates.up.sql (hunk 1 above)
Comment (axrs) [reply resources/migrations/20170222203502-aggregates.up.sql]: Correct. A single aggregate is a combination of multiple activities. Hence an array type :D

**Review by axrs — CHANGES_REQUESTED** [review]

**Comment by CalebMacdonaldBlack** [comment]
@axrs

**Review by axrs — APPROVED** [review]

**Pull request merged**